src/screens/HomeScreen.tsx

```diff
@@ -50,6 +50,7 @@ const styles = StyleSheet.create({
     justifyContent: 'space-evenly',
     alignItems: 'center',
     height: '100%',
+    paddingBottom: '10%',
```

Comment:
Werken met `%` is gevaarlijk. Je weet niet wat de 10% doet als het scherm groter staat ingesteld en op een té groot scherm is de padding dan enorm.

Reply:
padding bottom is weggehaald, had meer gebruikt vanwege de spacing tussen de buttons. maar dat is al gefixt vanwege homescreen pr